src/molecule/api.py

```diff
@@ -36,6 +36,12 @@ def append(self, item) -> None:
         super(UserListMap, self).append(item)
 
 
+class MoleculeRuntimeWarning(RuntimeWarning):
```

Comment:
I like this. I think we may want to also explore having some more specific warnings in the hierarchy. Like `IncompatibleMoleculeRuntimeWarning(MoleculeRuntimeWarning)` maybe.

Reply:
I sense some over-engineering odour here... but whatever, I will do it.